### DISCUSSION TITLE
Image is missing required "src" property

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#29365] [Help]
### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

v16.10.0

### What browser are you using?

Chromium, Firefox

### What operating system are you using?

Linux

### How are you deploying your application?

next dev

### Describe the Bug

`Error: Image is missing required "src" property. Make sure you pass "src" in props to the `next/image` component. Received: {}`

Already have look the other issues and idk where is the problem, also I have commented everything lines that has relation with <Image /> component and the error continues

I tried remove `node_modules/` - `.next/` and nothing

### Expected Behavior

Get my images :(

### To Reproduce

For example I saw on the other issues that fix it with a flag, and my only block code where possibly generate an error is here:

```
{item?.image && (
              <Image
                src={item?.image}
                alt={item?.productName}
                className="mx-auto"
                layout="fill"
              />
)}
```

But, from beginning my code looks like above, because my rest images are files server from `public/`  

I noticed that error was thrown when I put a `gif` file, i don't believe that a gif file be the error

**Comment by styfle** [comment]
Please provide the full code to reproduce, perhaps with https://codesandbox.io

Chances are, its a different image than the one you wrapped.

Here's the source code for that error:

https://github.com/vercel/next.js/blob/13893395dc6d97ecbbc4ed75f531fc9fc5902d2a/packages/next/client/image.tsx#L371-L377

**Reply by ghost** [reply]
I continue with this error :( and already I tried with everything, clear cache, node_modules, remove .next/ directory, remove all lines that has relation with Image component and doesn't works :(

Should I create a new project?

**Reply by styfle** [reply]
You could try finding all usages of `<Image>` and add a unique `quality` value to each one. Then when you get the error message, you'll see which one was missing `src`.

**Reply by ghost** [reply]
The error continues :'(

**Comment by rvramesh** [comment]
I believe you need to remove the `?` operator when assigning to `src` 

Can you try?

```
{item && item?.image && (
              <Image
                src={item.image}
                alt={item?.productName}
                className="mx-auto"
                layout="fill"
              />
)}
```

**Reply by ghost** [reply]
Noup doesn't works :/
In fact currently I have this line commented, so idk i what happens reality
